src/molecule_podman/playbooks/create.yml

```diff
@@ -178,6 +178,7 @@
       changed_when: true
 
     - name: Wait for instance(s) creation to complete
+      become: "{{ not item.item.rootless | default(omit) }}"
```

Comment:
I may be the ignorant here but my impression was that `become` was supposed to be used on looped `command` and not on the `async_status`, which I doubt it really needs root access to check already running tasks.